app/javascript/App/components/NewsfeedPage/CoinBody.js

```diff
@@ -24,7 +24,7 @@ const CoinBody = (props) => {
           <span className="ml2">({coin.get('symbol')})</span>
         </a>
         <div>
-          <WatchButton {...props} coin={coin} />
+          <WatchButton {...props} coin={coin} hasText={true} />
```

Comment:
omitting the value of an attribute will evaluate in descendant props, as true 

another way to write this:

```
<WatchButton {...props} coin={coin} hasText />
```